CLR/Core/CLR_RT_Memory.cpp

```diff
@@ -16,8 +16,6 @@ static int s_PreHeapInitIndex = 0;
 
 ////////////////////////////////////////////////////////////
 
-HAL_DECLARE_CUSTOM_HEAP( CLR_RT_Memory::Allocate, CLR_RT_Memory::Release, CLR_RT_Memory::ReAllocate );

```

Comment:
Are you entirely sure about that?!
The name refers to custom heap.
I have build this without this code and it went fine.
I have loaded the image on real hardware a run a test app. Creating and destroying objects. Checked the managed memory going up and down a seeing the GC kicking on. I admit it wasn't nothing fancy or exhaustive but it proved that the managed heap was working.